### PR TITLE
feat:quiz authentication added

### PR DESCRIPTION
**Pull request by tushargoyal812**
quiz-login
quiz-signup

### CODE REVIEW

File: quiz-signup/quiz-signup.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ShopShip/SignUp</title>
```
Comment (syed-shaida-hussain) [inline quiz-signup/quiz-signup.html]: change the title

**Review by syed-shaida-hussain — APPROVED** [review]
Change the title in login and signup . Otherwise looks good

File: quiz-login/quiz-login.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            <label for="" class="email-label">Email address</label>
+            <div>
+                <input type="text" class="login-input px-3">
+            </div>
+            </div>
+            <div class="pb-1">
+            <label for="" class="passwaord-label">Password</label>
+            <div>
+                <input type="password" class="login-input px-3">
+            </div>
```
Comment (vishalg8454) [inline quiz-login/quiz-login.html:27]: `for` attribute is missing.

File: quiz-signup/quiz-signup.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            <label for="" class="email-label">Email address</label>
+            <div>
+                <input type="text" class="signup-input px-3">
+            </div>
+            </div>
+            <div class="pb-1">
+            <label for="" class="passwaord-label">Password</label>
+            <div>
+                <input type="password" class="signup-input px-3">
+            </div>
```
Comment (vishalg8454) [inline quiz-signup/quiz-signup.html]: again, same problem.

**Review by vishalg8454 — APPROVED** [review]
Just minor change, rest all looks good.

**Pull request merged**